docs/source/federation/migrating-from-stitching.md

```diff
@@ -11,9 +11,17 @@ schema stitching, see [this blog post](https://blog.apollographql.com/apollo-fed
 
 ## Summary of steps
 
-You can (and should) migrate **incrementally** from schema stitching to Apollo Federation.
-To do so, you run an Apollo Server gateway _alongside_ your existing schema-stitching gateway and migrate the services that implement your data graph (**implementing services**)
-one at a time.
+This documentation describes a set of steps for migrating architecture **incrementally** from
```

Comment:
Use "article" instead of "documentation" when referring to an individual piece of content

Reply:
Or "guide" as we do in the intro currently